examples/Main.hs

```diff
@@ -1,8 +1,10 @@
 {-# LANGUAGE OverloadedStrings #-}
+-- OverloadedLists here causes things like forM_ [0..3] cause type ambiguity
```

Comment:
No, it doesn't. The reason why I mentioned this, is because in the docs for Clay ([Clay.Box](http://hackage.haskell.org/package/clay-0.13.3/docs/Clay-Box.html#g:1)), it is suggested to use `OverloadedLists` to type, for example, `boxShadow [none]` instead of `boxShadow $ fromList [none]` or `boxShadow . pure $ none`. However, this comes with a little problem, that now things like `forM_ [0..3] do ...` don't typecheck unless we specify type directly, like `forM_ ([0..3] :: [Integer]) do ...`. This is very minor, but I think it is worth mentioning in the docs.

Reply:
Ah, I understand now! Can you mention this caveat in the haddocks of `Clay.Box`?

Reply:
Sure, should I create another PR just for that?

Reply:
You can, but this one is fine as well.